odoo/http.py

```diff
@@ -542,6 +542,14 @@ def from_attachment(cls, attachment):
             self.last_modified = attachment.write_date
             self.size = len(self.data)
 
+        elif attachment.type == 'cloud_storage' and attachment.env['cloud.storage.provider']._get_configuration():
+            self.type = 'url'
+            info = attachment.env['cloud.storage.provider']._generate_download_info(attachment)
```

Comment:
I think it is inevitable. The cloud storage url with access token is generated dynamically and needs different treatment for max_age to avoid permanent HTTP cache
There are other 2 ideas I thought about but gave up.
1. when `attachment.url` should be used, call a method, e.g. `attachment.get_download_url()['url']` and we can override `get_url` for the cloud storage. But I think it makes the normal code tricky and more difficult to use. But maybe you prefer this solution?
2. still call `attachment.url`, but add the signature in `convert_to_record` for the url field. But our framework doesn't support this hack.